src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/DivInstruction.cs

```diff
@@ -189,186 +189,4 @@ public static Instruction Create(Type type)
             }
         }
     }
-
-    internal abstract class ModuloInstruction : Instruction
-    {
-        private static Instruction s_int16, s_int32, s_int64, s_UInt16, s_UInt32, s_UInt64, s_single, s_double;
-
-        public override int ConsumedStack => 2;
-        public override int ProducedStack => 1;
-        public override string InstructionName => "Modulo";
-
-        private ModuloInstruction() { }
-
-        private sealed class ModuloInt32 : ModuloInstruction
-        {
-            public override int Run(InterpretedFrame frame)
-            {
-                object l = frame.Data[frame.StackIndex - 2];
-                object r = frame.Data[frame.StackIndex - 1];
-                if (l == null || r == null)
-                {
-                    frame.Data[frame.StackIndex - 2] = null;
-                }
-                else
-                {
-                    frame.Data[frame.StackIndex - 2] = ScriptingRuntimeHelpers.Int32ToObject((int)l % (int)r);
-                }
-                frame.StackIndex--;
-                return 1;
-            }
-        }
-
-        private sealed class ModuloInt16 : ModuloInstruction
-        {
-            public override int Run(InterpretedFrame frame)
-            {
-                object l = frame.Data[frame.StackIndex - 2];
-                object r = frame.Data[frame.StackIndex - 1];
-                if (l == null || r == null)
-                {
-                    frame.Data[frame.StackIndex - 2] = null;
-                }
-                else
-                {
-                    frame.Data[frame.StackIndex - 2] = (short)((short)l % (short)r);
-                }
-                frame.StackIndex--;
-                return 1;
-            }
-        }
-
-        private sealed class ModuloInt64 : ModuloInstruction
-        {
-            public override int Run(InterpretedFrame frame)
-            {
-                object l = frame.Data[frame.StackIndex - 2];
-                object r = frame.Data[frame.StackIndex - 1];
-                if (l == null || r == null)
-                {
-                    frame.Data[frame.StackIndex - 2] = null;
-                }
-                else
-                {
-                    frame.Data[frame.StackIndex - 2] = (long)l % (long)r;
```

Comment:
I had something similar to that idea as part of #13467 but after seeing the effort instructions like this go to to shave off even the calls to `Pop()` and `Push()` most of them have, decided that adding a virtual call to each of them was flying in the face of that, and removed that commit.